app/views/merchant/merchant/show.html.erb

```diff
@@ -1 +1,6 @@
-<%= link_to "My Items", merchant_items_path %>
\ No newline at end of file
+<p>Order Numbers</p>
+<%@orders.each do |order|%>
```

Comment:
Should we be passing just the orders through to this view or would it be better if we passed the merchant